bigbluebutton-html5/imports/ui/components/settings/submenus/users/participants/component.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import Modal from 'react-modal';
+import Icon from '/imports/ui/components/icon/component';
+import Button from '/imports/ui/components/button/component';
+import BaseMenu from '../../base/component';
+import styles from '../../styles.scss';
+import Users from '/imports/api/users';
+import AuthSingleton from '/imports/ui/services/auth/index.js';
+
+
+export default class ParticipantsMenu extends BaseMenu {
+  constructor(props) {
+    super(props);
+  }
+
+  checkUserRoles(){
+    let user = Users.findOne({
+       userId: AuthSingleton.getCredentials().requesterUserId,
+     }).user;
+
+    return {
+     isPresenter: user.presenter,
+     role: user.role,
+    };
+  };
+
+  renderAdmin() {
+    return (
+      <div className={styles.full} role='presentation'>

```

Comment:
these items could probably be created inside a `.map`, this way you don't need to have all the code that is repeating itself. Maybe something like: 
```
 //(...)
  lockItems.map((lockItem, i) => this.renderLockItem(lockItem, i));
 }

renderLockItem(lockItem, i) {
    return (
      <div className={styles.row} role='presentation'>
        <label>
          <input 
            className={styles.checkboxOffset} 
            type="checkbox" 
            tabIndex={i}
            aria-labelledby={lockItem.ariaLabelledBy} 
            aria-describedby={lockItem.ariaDescribedBy} />
          {lockItem.label}
        </label>
        <div id={lockItem.ariaLabelledBy} hidden>{lockItem.ariaLabel}</div>
        <div id={lockItem.ariaDescribedBy} hidden>{lockItem.ariaDesc}</div>
      </div>
    );
  }
```